include/avif/avif.h

```diff
@@ -571,7 +571,6 @@ typedef struct avifIO
 
 avifIO * avifIOCreateMemoryReader(const uint8_t * data, size_t size);
 avifIO * avifIOCreateFileReader(const char * filename);
-avifIO * avifIOCreateFilePtrReader(FILE * f, avifBool closeOnDestroy);
```

Comment:
Could you also remove #include <stdio.h> from avif.h? That #include statement was added for this function.

Reply:
Done.